packages/ui/src/components/Button/Button.tsx

```diff
@@ -16,32 +16,33 @@ export function Button({
     disabled,
     children,
     contentClassName,
+    rounded,
     ...buttonProps
 }: Button.Props) {
-    const isIconOnly = Boolean(Icon) && !children;
-
     return (
         <button
             ref={forwardRef}
             type={type}
             className={twMerge(
-                'rounded flex items-center justify-center border border-transparent bg-transparent',
-                size === 'small' ? `p-3 label-medium` : `p-4 label-large`,
+                `${
+                    rounded ? 'rounded-full' : 'rounded'
+                } flex items-center justify-center border border-transparent bg-transparent focus:ring-4 focus-within:ring-4 focus:ring-accent-lighter focus-within:ring-accent-lighter`,
```

Comment:
It would be cleaner to place this tertiary in a separate string like you do with other conditional properties, that way it would be placed on a single line
```suggestion
                rounded ? 'rounded-full' : 'rounded',
                'flex items-center justify-center border border-transparent bg-transparent focus:ring-4 focus-within:ring-4 focus:ring-accent-lighter focus-within:ring-accent-lighter',
```